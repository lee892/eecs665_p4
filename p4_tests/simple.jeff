bool glob;
fn : () int main{
	int a;
	glob = a;
}
