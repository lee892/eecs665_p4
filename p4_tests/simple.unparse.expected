bool glob;
fn : () int main{
	int a;
	glob{bool} = a{int};
}
